core/src/test/java/io/lionweb/lioncore/java/testset/Invalid.java

```diff
@@ -0,0 +1,58 @@
+package io.lionweb.lioncore.java.testset;
+
+import static org.junit.Assert.assertThrows;
+
+import io.lionweb.lioncore.java.model.Node;
+import io.lionweb.lioncore.java.serialization.JsonSerialization;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+@RunWith(Parameterized.class)
+public class Invalid {
+  private final Path path;
+
+  @Parameterized.Parameters
+  public static Object[] inputFiles() {
+    Path integrationTests = findIntegrationTests();
+    Path basePath = integrationTests.resolve("testset").resolve("invalid");
+    try {
+      Object[] result =
+          Files.walk(basePath)
+              .filter(Files::isRegularFile)
+              .filter(f -> f.toString().endsWith(".json"))
+              .toArray();
+      return result;
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  public Invalid(Path path) {
+    this.path = path;
+  }
+
+  @Test
+  public void assertInvalid() {
+    assertThrows(
+        path.toString(),
+        RuntimeException.class,
+        () -> {
+          List<Node> nodes =
+              JsonSerialization.getStandardSerialization().unserializeToNodes(path.toFile());
+          System.out.println(nodes);
+        });
+  }
+
+  private static Path findIntegrationTests() {
+    String propertyValue = System.getenv("integrationTestingDir");
```

Comment:
I am not sure how I can pass an environment variable when running the tests from within the IDE.
Perhaps I would instead either download the files as part of the test, or expect them to be in a certain directory relative to the root of the project (e.g., `build/integrationTests`)

Reply:
Yes, I don't have a great idea for this either. Just implemented the next best thing besides hard-coded paths.